### ISSUE TITLE
compiler warning: missing set initializer

### ISSUE THREAD

**Issue by Wang-Yue** [cmus/cmus#529]
since commit  9536ecc2436f2d2587a8ad42c3b97aba700c4328 various files has this issue

for instance

```
coreaudio.c:864:9: warning: missing field 'set' initializer
      [-Wmissing-field-initializers]
        { NULL },
               ^
coreaudio.c:868:9: warning: missing field 'set' initializer
      [-Wmissing-field-initializers]
        { NULL },
               ^
```

**Comment by Wang-Yue** [comment]
@mahkoh  please fix

**Comment by mahkoh** [comment]
Fixed in #511
